src/components/List.js

```diff
@@ -0,0 +1,26 @@
+import ListItem from "./ListItem.js";
+
+export default function List({
+  setToDoBox,
+  listItems,
+  handleToggle,
+  handleDeleteToDo,
+  handleRenamingToDo,
+}) {
+  function renderList() {
+    return listItems.map((toDoBox) => {
+      return (
+        <ListItem
+          handleRenamingToDo={handleRenamingToDo}
+          setToDoBox={setToDoBox}
```

Comment:
You are passing this prop `setToDoBox` to the `ListItem` component, but that prop is not used in the `ListItem`component.